libraries/AP_Common/tests/test_expandingstring.cpp

```diff
@@ -12,6 +12,12 @@ TEST(ExpandingString, Tests)
     EXPECT_EQ(5u, test_string->get_length());
     EXPECT_FALSE(test_string->has_failed_allocation());
     EXPECT_TRUE(test_string->append("Test2\n", 6));
+    test_string->~ExpandingString();
```

Comment:
Uh - is it safe to deal with `test_string` (i.e. get_string) after you destroy this?

Reply:
Probably not  but how do you test a destructor? Overriding free() to get something from it?